Formula/periscope.rb

```diff
@@ -0,0 +1,52 @@
+class Periscope < Formula
+  desc "Organize and de-duplicate your files without losing data"
+  homepage "https://github.com/anishathalye/periscope"
+  url "https://github.com/anishathalye/periscope.git",
+    tag:      "v0.2.0",
+    revision: "d672bf60f4b59c1f54fa3c26aef75d0593615c40"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags",
+      "-s -w -X main.version=#{version} -X main.commit=#{stable.specs[:revision]}",
+      "-trimpath", "./cmd/psc"
```

Comment:
did you try `std_go_args`?

Reply:
`std_go_args` doesn't work. We need to explicitly pass certain ldflags so `psc version` can report the correct output. Other formulae like [goreleaser](https://github.com/Homebrew/homebrew-core/blob/master/Formula/goreleaser.rb#L20) do this as well. That formula tacks on `std_go_args` at the end, but we can't use that either, because we need to explicitly pass `./cmd/psc` (or `./...`) to build the program that's contained in a subdirectory; running a `go build` in the root directory doesn't actually build anything.

Reply:
I think some of them specify a path on top of the std go args

Reply:
`std_go_args` = `-trimpath -o /usr/local/Cellar/periscope/0.2.0/bin/periscope`. The `-trimpath` is fine (I'm manually specifying it), but we don't want the `-o [cut]/bin/periscope`, because the Periscope binary is called `psc`. That's why I have the build line, and then a separate `bin.install "psc"` line.